### PR TITLE
tuto 1

### PR DESCRIPTION
**Pull request by mloubout**
TLE tutos

### CODE REVIEW

File: publications.md [hunk 1]
```diff
@@ -8,6 +8,14 @@ permalink: /publications
 
 ### Paper publications
 
+* Mathias Louboutin, Philipp A. Witte, Michael Lange, Navjot Kukreja, Fabio Luporini, Gerard Gorman, and Felix J. Herrmann. 
+  Full-waveform inversion - Part 2: adjoint modeling. Submitted to The Leading Edge for the tutorial section on October 30, 2017., 2017.
+[[https://github.com/seg/tutorials-2018/tree/master/1801_FWI_Adjoint_modeling](https://github.com/seg/tutorials-2018/tree/master/1801_FWI_Adjoint_modeling)]
+
+* Mathias Louboutin, Philipp A. Witte, Michael Lange, Navjot Kukreja, Fabio Luporini, Gerard Gorman, and Felix J. Herrmann. 
+  Full-waveform inversion - part 1: forward modeling. Submitted to The Leading Edge for the tutorial section on October 30, 2017., 2017.
```
Comment (FabioLuporini) [inline publications.md]: the 'p' in "part 1" is lower here, the 'p' in "Part 2" is capital

**Comment by mloubout** [comment]
Can you add it yourself, no idea where in the order of author I should add yours (feel free to just push to this branch)

**Comment by mloubout** [comment]
All done, can we merge?

**Comment by FabioLuporini** [comment]
> Can you add it yourself

completely missed it. I see you did it, thanks. 

Merging...

**Comment by FabioLuporini** [comment]
Merged.

**Pull request merged**